docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:20.04
+RUN \
```

Comment:
Codacy found an issue: [This \P will be a regular 'P' in this context.](https://app.codacy.com/manual/ndelvalle/generator-api/pullRequest?prid=6300885)

Reply:
done